adafruit_azureiot/iot_mqtt.py

```diff
@@ -91,7 +91,7 @@ def device_twin_desired_updated(self, desired_property_name: str, desired_proper
         :param int desired_version: The version of the desired property that was updated
         """
 
-    def device_twin_reported_updated(self, reported_property_name: str, reported_property_value, reported_version: int) -> None:
+    def device_twin_reported_updated(self, reported_property_name: str, reported_property_value, reported_version: int,) -> None:
```

Comment:
Does this extra comma do anything?

Reply:
@jimbobbennett  Hrm, I didn't add them. These may have been added by `black`'ing the file.